scripts/pylintrc_default

```diff
@@ -290,6 +290,8 @@ good-names = i, j, k, ex, Run, _,
 
 # Regular expression matching correct method names
 # DEFAULT:  method-rgx=[a-z_][a-z0-9_]{2,30}$
+# RATIONALE:  mapping long API names onto connection methods
+method-rgx=[a-z_][a-z0-9_]{2,35}$
```

Comment:
It isn't an "unwritten rule" -- it was just the pattern I adopted when moving the API method-specific bits to the connection class.

I did have a thought about the pattern, based on looking at how the gRPC stuff is [organized]https://cloud.google.com/pubsub/reference/rpc/):  ISTM that we might split the API-specific methods into helper objects (matching the `IAMPolicy`, `Publisher`, and `Subscriber` entities on that page).  The "connection" object would then have properties which returned instances of those helpers.  In gRPC, those helpers correspond to the service-specific stubs.

@dhermes WDYT?